### PR TITLE
Include server-bid-cid in previous dump file

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: jobs/buildhub/to_kinto.py [hunk 1]
```diff
@@ -46,6 +46,11 @@ def fetch_existing(client, cache_file=PREVIOUS_DUMP_FILENAME):
     """Fetch all records since last run. A JSON file on disk is used to store
     records from previous run.
     """
+    cache_file = cache_file.format(
+        server="".join(c for c in client.session.server_url if c.isalnum()),
```
Comment (Natim) [inline jobs/buildhub/to_kinto.py]: How will that lookslike? `httpbuildhubstagemozawsnetv1` ?

Comment (leplatrem) [reply jobs/buildhub/to_kinto.py]: yep

Comment (Natim) [reply jobs/buildhub/to_kinto.py]: Maybe the hostname would be more readable?

Comment (leplatrem) [reply jobs/buildhub/to_kinto.py]: why would we need it to be more readable?

**Pull request merged**